admin_manual/configuration_files/file_sharing_configuration.rst

```diff
@@ -58,6 +58,9 @@ forcing them.
    `Wrong folder date when syncing 
    <https://github.com/owncloud/core/issues/7009>`_ for discussion of this.
 
+.. note:: There are more sharing options on config.php level available: 
+   <https://docs.nextcloud.com/server/latest/admin_manual/configuration_server/config_sample_php_parameters.html#sharing>
```

Comment:
Needs to be a relative link

Reply:
Can I link from the user manual to the admin manual? Ideally even to an anchor?